dapos/dapos_transport_http.go

```diff
@@ -53,6 +53,8 @@ func (this *DAPoSService) WithHttp() *DAPoSService {
 	services.GetHttpRouter().HandleFunc("/v1/page/{id}", this.unsupportedFunctionHandler).Methods("GET")
 	//analytical
 	services.GetHttpRouter().HandleFunc("/v1/queue", this.getQueueHandler).Methods("GET")
+	services.GetHttpRouter().HandleFunc("/v1/gossips", this.getGossipsHandler).Methods("GET")
+	services.GetHttpRouter().HandleFunc("/v1/gossips/{hash}", this.getGossipHandler).Methods("GET")
```

Comment:
We're currently REMOVING receipts as a stand-alone endpoint because they work better attached to the source transaction. Is there a reason the same isn't true of this model?

Reply:
I would like to keep the receipt endpoint for my big automated tests.  I already know my transaction data and want to poll for receipts until I get a status other than pending.
For Gossip, I think we ONLY need these endpoints while we are doing testing.  I personally don't think we need to expose gossip through http for the long term.  Probably want to keep a GRPC option for compiling statistics, unless we take another approach with that.
MHO